packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes                      from 'prop-types';
+import classNames                     from 'classnames';
+
+class ToggleSwitch extends Component {
```

Comment:
@mustofa-binary can you please use functional component here? to keep it the same for all components